### PR TITLE
This adds a file about calicut

### PR DESCRIPTION
**Pull request by rsatishwm**
Test

### CODE REVIEW

**Comment by hsinghwm** [comment]
# Tell me something about your city.. looks like there is not much to share??

File: calicut.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Testing github web commit
```
Comment (hsinghwm) [inline calicut.md:1]: Why no information here????????

**Comment by anjanikumarsingh** [comment]
Can you work on the bug please

**Comment by adityavipul** [comment]
Please delete this branch. Listen to instructions.

**Comment by anjanikumarsingh** [comment]
tell me what help is wanted

**Pull request merged**